AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -56,8 +56,8 @@ sallys-blog.de,kaiusa.com,hussel.de,dyson.com.tr,cherry.de,ba-computer.at,stoert
 ! ###gdpr-cookie-message
 demirorenpetrol.com.tr,onurmarket.com,mediamarkt.es,veren.org,yunusonline.com,lightnovelworld.com,burgerking.hu,sparktoro.com,oem-parts.hu,teamspeak.com###gdpr-cookie-message
 !
-digitalo.de#$?#body > .reveal-overlay:has(> #cc_info) { display: none !important; }
-digitalo.de#$#.is-reveal-open { overflow: visible !important; }
+voelkner.de,digitalo.de#?#body > .reveal-overlay:has(> #cc_info)
```

Comment:
@BlazDT if I'm not wrong ext-CSS rule is enough here because it don't work in content blockers

Reply:
Without overflow rule in digitalo.de scrolling was not possible, therefore In added it to it.

Reply:
I mean we don't have to add `{ display: none !important; }` to the .reveal-overlay rule =)

Reply:
I mean we don't have to add { display: none !important; } to the .reveal-overlay rule =) I've already changed it in commit.

Reply:
yep